test/fixtures/executions.yml

```
@@ -0,0 +1,6 @@
+live_ls:
+  script: ls
+  server: atahualpa
+  status: <%= Execution.statuses[:running] %>
+  started_at: <%= Time.zone.now %>
```

Esto debería ser:

```suggestion
  started_at: <%= Time.zone.now.to_s :db %>
```

Más académicamente correcto =P